web/locales/fr.json

```diff
@@ -67,1094 +109,1363 @@
+    },
+    "data": {
+        "groupName": "Nos données",
+        "dataOrigins-question": "Comment obtenez-vous vos données ?",
+        "dataOrigins-answer": "Nous calculons toutes nos données à partir de données accessibles au public, publiées par les opérateurs de réseaux électriques, les agences officielles et autres. Vous pouvez cliquer sur une zone pour voir plus de détails sur les origines de ces données.",
+        "dataDownload-question": "Puis-je télécharger vos données?",
+        "dataDownload-answer": "Vous pouvez acheter un accès à toutes nos données dans notre <a href=\"https://data.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral\" target=\"_blank\">base de données</a>.",
+        "dataIntegration-question": "Puis-je intégrer vos données en temps réel dans mon application ou mon appareil?",
+        "dataIntegration-answer": "Oui! Nous vendons l'accès à l'API <a href=\"https://api.electricitymap.org?utm_source=electricitymap.org&utm_medium=referral\" target=\"_blank\">en temps réel</a>, qui inclut les prévisions pour l'avenir."
+    },
+    "aboutUs": {
+        "groupName": "À propos de nous",
+        "whoAreYou-question": "Qui êtes-vous ?",
+        "whoAreYou-answer": "La carte de l’électricité est développée et gérée par <a href=\"https://www.tmrow.com/\" target=\"_blank\">Tomorrow</a>, une petite entreprise franco-danoise. Notre objectif est d'aider l'humanité à atteindre un état d'existence durable en quantifiant et en rendant largement accessible l'impact sur le climat des choix quotidiens que nous faisons.",
+        "feedback-question": "Puis-je vous faire des commentaire ?",
+        "feedback-answer": "Faites donc ! Remplissez <a href=\"https://docs.google.com/forms/d/e/1FAIpQLSc-_sRr3mmhe0bifigGxfAzgh97-pJFcwpwWZGLFc6vvu8laA/viewform?c=0&w=1\" target=\"_blank\">notre formulaire de réponse</a> ou <a href=\"mailto:hello@tmrow.com\">envoyez-nous un email</a> !",
+        "contribute-question": "Puis-je contribuer à votre projet ?",
+        "contribute-answer": "Bien sûr ! Electricity Map est un projet open-source, rendu possible par nos contributeurs bénévoles. Si vous souhaitez contribuer au développement de la carte, en ajoutant des sources de données pour de nouvelles zones, en ajoutant de nouvelles fonctionnalités ou en corrigeant des bugs, n'hésitez pas à nous rejoindre sur notre <a href=\"https://github.com/corradio/electricitymap/\" target=\"_ blank\"> github </a>.",
+        "workTogether-question": "Pouvons-nous travailler ensemble ?",
+        "workTogether-answer": "Nous recherchons activement de nouvelles opportunités de collaboration. <a href=\"mailto:hello@tmrow.com\">Envoyez-nous un email</a> !",
+        "disclaimer-question": "Avertissement",
```

Comment:
The comma here is causing the build to fail, however the error from this is being swallowed by something else.